platform/platform-paxweb-jettyconfig/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -75,6 +77,16 @@
 
     <service interface="org.osgi.framework.hooks.service.EventListenerHook" ref="filterInjector"/>
 
-    <reference-list id="filterList" availability="optional" interface="javax.servlet.Filter" />
+    <service id="DoSFilter" interface="javax.servlet.Filter" ranking="100">
+        <service-properties>
+            <entry key="filter-name" value="DoSFilter"/>
+            <entry key="urlPatterns" value="/*"/>
+            <entry key="init.maxRequestsPerSec" value="1000"/>
+            <entry key="init.delayMs" value="-1" />
```

Comment:
❓ I figured we would want to add a delay to the request that go over the 1000 requests per sec.

Reply:
Having a delay of -1 tells the filter to just reject any request after the 1000th in a second. I figured that if we just delay the requests they could still bog down the system. I thing 1000 requests per sec is generous enough that we can just reject requests over that but this could be a point of further discussion. I'd like for others to share their thoughts.